app/src/app/components/contents/PacketGroup/PacketTable.tsx

```diff
@@ -15,6 +15,7 @@ export const PacketTable = () => {
   const { packetName } = useParams();
   const [pageNumber, setPageNumber] = useState(0);
 
+  // Rename useGetPacketGroups to useGetPacketGroupPages, and packetGroups to packetGroupPages?
```

Comment:
Why?

Reply:
When you iterate over those things, the elements aren't packet groups, they are pages of packet groups. So later in the file we have eg 'packetGroups.first' which looks like it means 'the first packet group' but actually means 'the first page of packet groups'.

Reply:
Oh, I mistook this file for being `PacketGroupSummaryList.tsx`. In fact, each item returned by useGetPacketGroups (each so-called packetGroup) is a _page of packets_, not a packet group nor a page of packet groups. So these things should be renamed to useGetPacketPages and packetPages respectively.

Reply:
or more simply, `useGetPackets` and `pages`